MvvmCross/Binding/Extensions/MvxEnumerableExtensions.cs

```diff
@@ -84,5 +86,25 @@ public static object ElementAt(this IEnumerable items, int position)
 
             return enumerator.Current;
         }
+
+        public static IEnumerable Filter(this IEnumerable items, Func<object, bool> predicate)
+        {
+            if (items == null)
+                return null;
```

Comment:
Would it be better to return empty array instead of null - it'll cause few breaks for developers that fail to do a null check on the returned result